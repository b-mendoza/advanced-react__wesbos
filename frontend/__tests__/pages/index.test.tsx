import { render, screen } from '@testing-library/react';

import IndexPage from '@/pages/index';

describe('Tests to IndexPage component', () => {
  it('Takes a Snapshot of the `IndexPage` component', () => {
    const { container } = render(<IndexPage />);

    expect(container).toMatchSnapshot();
  });

  it('Renders initial JSX from `IndexPage` component', () => {
    render(<IndexPage />);

    const heading = screen.getByRole('heading', { name: /hello world/i });

    expect(heading).toBeInTheDocument();
  });
});
