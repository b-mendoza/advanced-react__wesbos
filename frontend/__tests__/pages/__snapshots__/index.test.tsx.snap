// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests to IndexPage component  1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;

exports[`Tests to IndexPage component Takes a Snapshot of the \`IndexPage\` component 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
